services.yaml

```diff
@@ -5177,6 +5340,39 @@ definitions:
       plural_suffix:
         type: string
         description: plural suffix/descriptor for custom units
+  JiraProject:
+    type: object
+    properties:
+      name:
+        type: string
+        description: project name
+        example: Engineering
+      issue_types:
+        type: array
+        items:
+          type: string
+          example: "Bug"
+  JiraField:
```

Comment:
This isn't the correct jira field. Look for `IssueTypeIssueCreateMetadata` in Atlassian's docs.

Reply:
updated